src/PluginRegistrationDialog.cpp

```diff
@@ -108,22 +105,19 @@ void PluginRegistrationDialog::PopulateOrExchange(ShuttleGui &S)
    auto& moduleManager = ModuleManager::Get();
    for(auto& [name, provider] : moduleManager.Providers())
    {
-      auto symbol = provider->GetSymbol();
-      if(std::find(
-         hiddenProviders.begin(),
-         hiddenProviders.end(), symbol) != hiddenProviders.end())
-      {
-         continue;
-      }
-      if(!symbol.empty())
+      //Use same name as in prefs
+      auto familySymbol = provider->GetOptionalFamilySymbol();
+      if(!familySymbol.empty())
+         pluginProviderNames.push_back(familySymbol.Msgid());
+      else
       {
-         auto it = nameSubstitutions.find(symbol.Msgid());
-         if(it != nameSubstitutions.end())
+         auto it = extraProviders.find(provider->GetSymbol().Msgid());
+         if(it != extraProviders.end())
             pluginProviderNames.push_back(it->second);
          else
-            pluginProviderNames.push_back(symbol.Msgid());
-         mPluginProviderIDs.push_back(PluginManager::GetID(provider.get()));
+            continue;
```

Comment:
With this change it is not guaranteed anymore that `pluginProviderNames` and `mPluginProviderIds` have equal size anymore, is it? Then it is used in
```cpp
void PluginRegistrationDialog::OnTypeFilterValueChanged(wxCommandEvent& evt)
{
   const auto index = evt.GetInt();
   if(index >= 0 && index < mPluginProviderIDs.size())
      mPluginsModel->SetFilterType(mPluginProviderIDs[index]);
}
```
and I haven't yet understood what the consequences could be.

Reply:
All good, I didn't realize that `continue` also skipped the call to `mPluginProviderIDs.push_back(PluginManager::GetID(provider.get()));`.